packages/redux/src/checkout/selectors.ts

```diff
@@ -762,3 +764,47 @@ export const getOperationsPagination = createSelector(
     return operations.result;
   },
 );
+
+/**
+ * Returns the loading status for the order item removing operation.
+ *
+ * @param state - Application state.
+ *
+ * @returns Order operation fetch process loading status.
+ */
+export const isRemoveOrderItemLoading = (state: StoreState): boolean =>
+  getRemoveOrderItem(state.checkout).isLoading;
+
+/**
+ * Returns the error for the order item removing operation.
+ *
+ * @function
```

Comment:
In the previous function, the TsDoc doesn't have the "@function" annotation, and I think you don't need it in these ones as well.

Reply:
🤦 ✅